### PR TITLE
feat(app): add skip to the content link

### PR DESCRIPTION
**Pull request by grzegorzpokorski**
PR improve accessibility by adding skip to the content link.

### CODE REVIEW

File: apps/app/src/components/SkipToTheContent.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const SkipToTheContent = () => (
+	<a
```
Comment (grzegorzpokorski) [inline apps/app/src/components/SkipToTheContent.tsx:2]: I need confirmation that I used proper tag - I'm not sure that we can't achieve that functionality by using the `Link` from `next/link`.

When I wanted to use `Link` component from `next/link` I realized that when I click on a link which contains for eg. `#element` at the end of the href attribute it does not cause scrolling page to the pointed element so I used native html `a` tag which works as expected.

Comment (grzegorzpokorski) [reply apps/app/src/components/SkipToTheContent.tsx:2]: I found something about my concerns: https://github.com/vercel/next.js/issues/44295

File: apps/app/src/components/SkipToTheContent.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+		href={`#main-content`}
+		className="fixed -top-[100%] z-50 m-4 border-white bg-violet-900 p-4 px-8 text-white focus:top-0 focus:border-2 focus:border-dashed"
+	>
+		Przejdź do treści
+	</a>
+);
```
Comment (mmiszy) [inline apps/app/src/components/SkipToTheContent.tsx:8]:
```suggestion
export const SkipToTheContent = () => {
	// @todo replace with `Link` when https://github.com/vercel/next.js/issues/44295 is fixed
	return (
		<a
			href={`#main-content`}
			className="fixed -top-[100%] z-50 m-4 border-white bg-violet-900 p-4 px-8 text-white focus:top-0 focus:border-2 focus:border-dashed"
		>
			Przejdź do treści
		</a>
	);
};
```

**Review by mmiszy — CHANGES_REQUESTED** [review]

**Comment by mmiszy** [comment]
Konflikty.

**Review by mmiszy — APPROVED** [review]

**Pull request merged**